distributions/standalone/assembly-whitelist.txt

```diff
@@ -379,7 +379,8 @@
 /mule-standalone-${productVersion}/lib/opt/reflections-0.9.9.jar
 /mule-standalone-${productVersion}/lib/opt/relaxngDatatype-20020414.jar
 /mule-standalone-${productVersion}/lib/opt/rhino-1.7R4.jar
-/mule-standalone-${productVersion}/lib/opt/rome-0.9.jar
+/mule-standalone-${productVersion}/lib/opt/rome-1.5.0.jar
```

Comment:
I need to update the library to able to implement my own parser.
Previous versions don't allow that.